cmd/server/handlers/handlers.go

```diff
@@ -6,22 +6,25 @@ import (
 	"strconv"
 	"strings"
 
-	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/storage/memstorage"
+	"evgen3000/go-musthave-metrics-tpl.git/cmd/server/storage"
```

Comment:
[CHANGE REQUEST] Please improve your layout.
https://github.com/golang-standards/project-layout

Reply:
What do you mean? I guess, this layout really simple and understandable. Could you tell me what is wrong?

Reply:
The handlers package should be in the `internal` package. The `cmd` package should be simple.